fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryAggregateIndexPlan.java

```diff
@@ -260,13 +260,11 @@ public RecordQueryAggregateIndexPlan translateCorrelations(@Nonnull final Transl
 
         final var translatedIndexPlan = indexPlan.translateCorrelations(translationMap,
                 shouldSimplifyValues, translatedQuantifiers);
-        final var newResultValue = resultValue.translateCorrelations(translationMap, shouldSimplifyValues);
-        final var newGroupByResultValue = groupByResultValue.translateCorrelations(translationMap, shouldSimplifyValues);
 
         // this is ok as there are no new quantifiers
-        if (translatedIndexPlan != indexPlan || newResultValue != resultValue || newGroupByResultValue != groupByResultValue) {
-            return new RecordQueryAggregateIndexPlan(translatedIndexPlan, recordTypeName, toRecord, newResultValue,
-                    newGroupByResultValue, constraint);
```

Comment:
Why is it okay to remove these translations? Overall, I'm not sure I understand what the changes to this class are getting at, if I'm honest

Reply:
So there are two attributes (both `Values`) that are in this class which shouldn't be in here as the nothing in here needs it. So they are planner breadcrumbs. And worse, these `Values` are values that come from the match candidate that this index plan is associated with... together with quantifier references that make sense within the candidate but not within the query. So translating it here is outright wrong. And since these values are redundant at best, incorporating them into the hashing and the equality logic is wrong as well.

Reply:
This raises the question as to why they cannot be removed. They can and they should. But that would require us to bump the plan hash version.

Reply:
Do we have an issue about this? It would be nice if there were a link out to that issue in here, which may aid future readers of this code block